src/core/api/clock.ts

```diff
@@ -292,11 +301,16 @@ function getStalledStatus(
   if (shouldUnstall === true) {
     return null;
   } else if (shouldStall === true || prevStalled !== null) {
-    let reason : "seeking" | "not-ready" | "buffering";
+    let reason : "seeking" | "not-ready" | "buffering" | "internal-seek";
     if (currentState === "seeking" ||
-        currentTimings.seeking ||
         prevStalled !== null && prevStalled.reason === "seeking") {
       reason = "seeking";
+    } else if (currentTimings.seeking &&
+        currentState === "internal-seeking" || 
+        prevStalled !== null && prevStalled.reason === "internal-seek") {
```

Comment:
Need parenthesis because I'm not sure it always return the expected result:
```
false && true || true && true => true
```
Is it what is wanted?